compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -355,13 +369,14 @@ void RecordMinMax::profileData(const std::string &mode, const std::string &input
     const auto input_nodes = loco::input_nodes(_module->graph());
     const auto num_inputs = input_nodes.size();
 
-    for (int32_t record_idx = 0; record_idx < num_records; record_idx++)
+    std::vector<std::vector<std::vector<char>>> vector_input_data(num_records);
+
+    // Read inputs to vector_input_data
+    for (int i = 0; i < num_records; ++i)
     {
-      if (num_inputs != importer.numInputs(record_idx))
+      if (num_inputs != importer.numInputs(i))
         throw std::runtime_error("Wrong number of inputs.");
 
-      std::cout << "Recording " << record_idx << "'th data" << std::endl;
```

Comment:
this log was added for CI not to stop for long running with no response from the program.
I'm not sure current CI requires this but I don't want this log to be removed.

As I wrote in previous comment, this PR is quite hard to read from multiple purpose of changes and especially removal of the log. Ad I am not willing to approve this by this.